src/types/Account/Account.spec.js

```diff
@@ -90,12 +74,9 @@ describe('Account - class', function suite() {
   it('should forward events', function (done) {
     const mockWallet = mocks.wallet;
     const account = new Account(mockWallet, { injectDefaultPlugins: false });
-    account.init(mockWallet).then(async ()=>{
-      await account.on(EVENTS.BLOCKHEADER, ()=>{
-        done();
-      });
-      account.storage.importBlockHeader(blockHeader);
-    })
-
+    account.on(EVENTS.STARTED, ()=>{
+      done();
+    });
+    account.init(mockWallet);
```

Comment:
Is it still an async function that you can wait?

Reply:
Right now, READY, STARTED and INITIALIZED event are still event that are thrown by initializeAccount. 
The change we made is that await .isReady() is not needed as instead await account.init() provide that features.